cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_create.spec.js

```diff
@@ -91,6 +91,96 @@ if (Cypress.env('WORKSPACE_ENABLED')) {
           cy.checkWorkspace(workspaceId, expectedWorkspace);
         });
       });
+
+      it('should successfully create a workspace from home page', () => {
+        cy.deleteWorkspaceByName(workspaceName);
+        miscUtils.visitPage('app/workspace_initial');
+        cy.getElementByTestId(
+          'workspace-initial-card-createWorkspace-button'
+        ).click({
+          force: true,
+        });
+        cy.getElementByTestId(
+          'workspace-initial-button-create-observability-workspace'
+        ).click({
+          force: true,
+        });
+        cy.contains('Observability')
+          .first()
+          .closest('.euiCheckableCard-isChecked')
+          .should('exist');
+
+        miscUtils.visitPage('app/workspace_initial');
```

Comment:
Why we need to visit initial page twice?

Reply:
to check if both these buttons could navigate to the create page
![Screenshot 2024-11-13 at 11 39 40](https://github.com/user-attachments/assets/92cc7493-7b8f-4f3e-a1d4-c3cf1a10a2f0)

Reply:
I don't think the workspace create integration test should cover this part. But it's OK. We can keep this.